integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala

```diff
@@ -53,6 +53,8 @@ object CompareResults {
         (spark.read.csv(conf.input1()), spark.read.csv(conf.input2()))
```

Comment:
Similarly lines https://github.com/NVIDIA/spark-rapids/pull/1545/files#diff-bc99ae208fb44d4abcf91b6e5b44515fbebeb90367c5f9eb1719e28226d22bf9R51-R68
can be 
```scala
    val format = spark.read.format(conf.inputFormat())
    BenchUtils.compareResults(
      format.load(conf.input1()),
      format.load(conf.input2()),
      conf.inputFormat(),
      conf.ignoreOrdering(),
      conf.useIterator(),
      conf.maxErrors(),
      conf.epsilon())
  }
```

Reply:
updated.